docs/system/hw-sw-mapping.rst

```diff
@@ -1,4 +1,28 @@
 Hardware/Software Mapping
 ===========================================
 
-by Evan Christopher
\ No newline at end of file
+The diagram above shows the hardware and software mapping of the system.
+Themis is used by tutors on an iPad. Feedback suggestions are provided by the Assessment subsystem within the app which depends 
+on the Submissions Subsystem of the Artemis Server and the Automatic Feedback Subsystem of the Themis-ML Server.
+Communication between nodes are facilitated by REST API.
+Please refer to `Artemis docs` and `Themis-ML` for more details regarding its deployment.
+
+Artemis' Submissions Subsystem depends on an Exercise Subsystem within the server, 
+which provides service for managing programming exercises and their configuration. 
+This subsystem in turn requires Themis' Automatic Feedback Subsystem, 
+which provides a tutor's feedback for a given submission. 
+
+Themis' Assessment Subsystem also requires the Automatic Feedback Subsystem of the Themis-ML server in order to provide feedback suggestions to the tutor.
+
+
+.. figure:: ../images/deployment_diagram.png
+   :scale: 100%
+   :alt: Deployment diagram of Themis
+   :align: center
+   :width: 100%
+
+   *Deployment overview*
+
+
+.. _Artemis docs: https://docs.artemis.cit.tum.de/dev/system-design/#deployment
+.. _Themis-ML: TODO
```

Comment:
https://ios2223cit.ase.cit.tum.de/ ?
or https://github.com/ls1intum/Themis-ML
or a link to the subpage within the docs (https://ls1intum.github.io/Themis/admin/deployment/ ?)

Reply:
I will use the link to the subpage then 👍